packages/ui/src/Stepper/StepAction.tsx

```diff
@@ -10,7 +10,7 @@ import styled, { css } from 'styled-components'
 import { isInProgress, taskStepStatusStyles } from './helpers'
 import Button from 'ui/src/Button'
 import Spinner from 'ui/src/Spinner/Spinner'
-import StepModal from 'ui/src/Stepper/StepModal'
+import { StepModal } from 'ui/src/Stepper'
```

Comment:
Huh, how did your tool not see `import * as React from 'react'` which can be removed as well?

Reply:
This opened a whole can of worms, sorry for the large diff :skull: